src/index.js

```diff
@@ -184,9 +184,12 @@ class LazyLoad extends Component {
 
     if (this.props.overflow) {
       const parent = scrollParent(ReactDom.findDOMNode(this));
-      if (parent && parent.getAttribute(LISTEN_FLAG) === null) {
-        parent.addEventListener('scroll', finalLazyLoadHandler);
-        parent.setAttribute(LISTEN_FLAG, 1);
+      if (parent) {
```

Comment:
`getAttribute` returns a string. without it, `1 + "1" === "11"`. had a fun journey back through type coercion while debugging this 😅